### ISSUE TITLE
An accessible Survey online.

### ISSUE THREAD

**Issue by stacylorraineprendeville** [FundacionParaguaya/stoplight-web#38]
How do we ensure the life map survey online is accessible?

1. Anything that isn’t text-based (images, videos) needs to be described in words using the appropriate tags and markup. The ALT text (alternative text for images) you use should be meaningful without being overly verbose. 
Review and Create amends

2. Transcripts for video and audio - If you use video, it should be accompanied by an accurate transcript (not just a machine-generated transcript; these are often wildly inaccurate). If you use YouTube to host your videos it’s easy to add captions using amara.org.

3. The markup on the page should flow logically so that it can be interpreted correctly as it is read aloud or converted into braille. Your heading should be labelled H1, with subheadings H2, H3 and so on. Don’t break convention. - THIS IS GOING TO BE AN ISSUE!

4. Contrast and Colours adjustable - The colour of your font should contrast with the page background so that it’s easy to read. Avoid using a pale shade of pink with red text, for example. 
REVIEW DESIGN

5. Each page should have clear headings, tagged with the right HTML tags, to help the user move around a page without necessarily clicking.

6. The page structure should stay roughly the same from section to section to help the user familiarise themselves. - HERE WE FAIL

7. Designers should consider eye tracking technology and touchscreens, ensuring there is open space between navigational elements and links.
TO REVIEW

8. Hyperlinks should be paired with descriptive anchor text, rather than generic phrases like ‘click here’ or ‘read more’.

9. Resizeable text (via CTRL +)  users can press the CTRL key and the + key together to enlarge the font size

10. Responsive design

11.  Form labels - Make sure fields are labelled and error messages make sense. Check you’ve labelled your buttons properly, too. Instead of ‘submit’, you may wish to rename the button to ‘send this email now’.
TO REVIEW

10. Screen Readers
The best way to test your content and structure is to have a screen reader read the pages out to you. You’ll still have a slightly skewed opinion of the results, given that you’ve seen what the page looks like already, but the way it’s interpreted may still surprise you.
There’s a list of screen readers on Wikipedia. We like ChromeVox and Fire Vox since they can be quickly added to your browser, but there are plenty of others to choose from. If you don’t have time to listen to the page, try the Fangs extension: it shows you a plain text version.
TO REVIEW

11. Toolbars - For testing as you code, a toolbar is a great way to quickly check you’re working to the correct guidelines. Again, there are plenty of options, including the Accessibility Evaluation Toolbar for Firefox and the Web Accessibility Toolbar for the Opera browser.

12. Web-Based Checkers - Quickly evaluate your entire site with a web-based accessibility checker like the web accessibility evaluation tool, or WAVE. It highlights structural problems, and it can even detect potential issues with contrast. If you’ve never bothered to check your site’s accessibility, WAVE makes it very quick and simple to get a basic idea of what you need to work on.

13. Keyboard accessibility - Check whether it is possible to navigate with the TAB and RETURN keys.

13. Desktop Tools - Although less common, desktop tools still have their place. One very clever Windows application is EveryEye, a simple programme that shows you what your website looks like to someone with a visual impairment, an elderly person or someone who is colourblind.

14. Add Sitemap

https://www.websitetooltester.com/en/blog/website-accessibility-for-disabled-users/

**Comment by gigerbytes** [comment]
Hey @stacylorraineprendeville is this a spike? how do we proceed with this?

**Comment by stacylorraineprendeville** [comment]
@gigerbytes I cannot do anything on this until we have a working survey. Then I will need to run a report on the work.

**Comment by stacylorraineprendeville** [comment]
This will require a duplicate issue within the FP PSP Front End Server

**Comment by Danman7** [comment]
I'm converting this into an epic and will add smaller issues for Pavel.
